test/images/agnhost/netexec/netexec.go

```diff
@@ -642,3 +649,21 @@ func assertNoError(err error, detail string) {
 		log.Fatalf("Error occurred: %s:%v", detail, err)
 	}
 }
+
+func parseAddresses(addresses string) []string {
+	if addresses == "" {
+		return []string{bindToAny}
+	}
+	// Using a set to remove duplicates
+	res := make([]string, 0)
+	split := strings.Split(addresses, ",")
+	for _, address := range split {
+		netAddr := net.ParseIP(address)
+		if netAddr == nil {
+			log.Printf("Invalid address %s", address)
+		}
+		res = append(res, address)
+	}
+	set := sets.NewString(res...)
+	return set.List()
```

Comment:
set := sets.NewString(res...)
	split := strings.Split(addresses, ",")
	for _, address := range split {
		netAddr := net.ParseIP(address)
		if netAddr == nil {
			log.Printf("Invalid address %s", address)
                        # or we fail here to avoid failing later
                        # or we swallow it with a continue
		}
		set.Insert(address)
	}
	return set.List()

Reply:
Done, went with the swallowing